src/main/java/mate/jdbc/model/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.jdbc.model;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.util.ConnectionUtil;
+import mate.jdbc.util.DataProcessingException;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturers(name,country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturersStatement =
+                        connection.prepareStatement(insertManufacturerRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturersStatement.setString(1, manufacturer.getName());
+            createManufacturersStatement.setString(2, manufacturer.getCountry());
+            createManufacturersStatement.executeUpdate();
+            ResultSet generateKeys = createManufacturersStatement.getGeneratedKeys();
+            if (generateKeys.next()) {
+                Long id = generateKeys.getObject(1,Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer to DB "
+                    + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturersStatement =
+                        connection.prepareStatement(getRequest)) {
+            getManufacturersStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturersStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getByResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer: id="
+                    + id + " from DB", e);
+        }
+        return null;
```

Comment:
```suggestion
        return Optional.empty();
```

It's bad practice to return `null`